relayer-engine/src/listener/missedVaaFetching.ts

```diff
@@ -136,6 +144,8 @@ async function tryFetchAndConsumeVaa(
       emitterAddress,
       sequence.toString(),
       { transport: grpcWebNodeHttpTransport.NodeHttpTransport() },
+      3000,
+      2,
```

Comment:
3 second timeout to query for VAA. 2 max retries. 

Should we make this into a setting? I hardcoded to follow the policy of don't make it over-configurable but maybe there's merit here?

Reply:
ooof that is a bug in my opinion, the delay should be at the end of the loop.